pkg/resolution/resolver/git/resolver_test.go

```diff
@@ -481,33 +517,30 @@ func TestResolve(t *testing.T) {
 				expectedStatus = tc.expectedStatus.DeepCopy()
 
 				if tc.expectedErr == nil {
-					reqParams := make(map[string]string)
-					for _, p := range request.Spec.Params {
-						reqParams[p.Name] = p.Value.StringVal
-					}
+					// status.annotations
 					if expectedStatus.Annotations == nil {
 						expectedStatus.Annotations = make(map[string]string)
 					}
 					expectedStatus.Annotations[resolutioncommon.AnnotationKeyContentType] = "application/x-yaml"
-					switch {
-					case tc.useNthCommit > 0:
-						expectedStatus.Annotations[AnnotationKeyRevision] = commits[plumbing.Master.Short()][tc.useNthCommit]
-					case tc.revision == "" && reqParams[urlParam] != "":
-						expectedStatus.Annotations[AnnotationKeyRevision] = plumbing.Master.Short()
-					case tc.revision == "":
-						expectedStatus.Annotations[AnnotationKeyRevision] = tc.config[defaultRevisionKey]
-					default:
-						expectedStatus.Annotations[AnnotationKeyRevision] = tc.revision
-					}
-					expectedStatus.Annotations[AnnotationKeyPath] = reqParams[pathParam]
+					expectedStatus.Annotations[AnnotationKeyRevision] = revisionToSHA[tc.args.revision]
+					expectedStatus.Annotations[AnnotationKeyPath] = tc.args.pathInRepo
 
-					if reqParams[urlParam] != "" {
-						expectedStatus.Annotations[AnnotationKeyURL] = reqParams[urlParam]
+					if tc.args.url != "" {
+						expectedStatus.Annotations[AnnotationKeyURL] = tc.args.url
 					} else {
-						expectedStatus.Annotations[AnnotationKeyOrg] = reqParams[orgParam]
-						expectedStatus.Annotations[AnnotationKeyRepo] = reqParams[repoParam]
+						expectedStatus.Annotations[AnnotationKeyOrg] = tc.args.org
+						expectedStatus.Annotations[AnnotationKeyRepo] = tc.args.repo
 						expectedStatus.Annotations[AnnotationKeyURL] = fakeClone
 					}
```

Comment:
Shouldn't this data always be the same? 🤔

Reply:
theoretically yes, args in the test cases are using those constants, and I used the value from test case to be consistent.
I just changed to the constant. But we cannot omit the if-else because annotation keys are different between scm cloning and anonymous cloning. Please let me know if you wanted to suggest something else :D